WordPress/Classes/Services/CommentService+Likes.swift

```diff
@@ -71,6 +74,25 @@ extension CommentService {
         return [LikeUser]()
     }
 
+    /**
+     Fetches a list of users from Core Data that liked the post with the given IDs after the given Date.
+     
+     @param commentID   The ID of the comment to fetch likes for.
+     @param siteID      The ID of the site that contains the post.
+     @param after       Filter results to likes after this Date.
+     */
+    func likeUsersFor(commentID: NSNumber, siteID: NSNumber, after: Date) -> [LikeUser] {
+        let request = LikeUser.fetchRequest() as NSFetchRequest<LikeUser>
+        request.predicate = NSPredicate(format: "likedSiteID = %@ AND likedCommentID = %@ AND dateLiked > %@", siteID, commentID, after as CVarArg)
```

Comment:
`AND dateLiked > %@",`
I think this would need to be `AND dateLiked < %@",` to get likers who liked later the value of `after`.